hkmc2DiffTests/src/test/scala/hkmc2/JSBackendDiffMaker.scala

```diff
@@ -48,6 +49,10 @@ abstract class JSBackendDiffMaker extends MLsDiffMaker:
     case ReplHost.Result(msg) =>
       if msg.startsWith("Uncaught") then output(s"Failed to load runtime: $msg")
     case r => output(s"Failed to load runtime: $r")
+    h.execute(s"const $termNme = (await import(\"${termFile}\")).default;") match
```

Comment:
This risks going slowing down every single diff-test, even thoguh very few use the quoting infrastructure. Can we gate this behind a flag?

Reply:
I've added a `:qq` flag for it.